OpenRA.Test/OpenRA.Test.csproj

```diff
@@ -25,27 +25,39 @@
     <Reference Include="Eluant, Version=1.0.5229.27703, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\thirdparty\Eluant.dll</HintPath>
-      <Private>False</Private>
+      <Private>True</Private>
```

Comment:
@pchote I managed to get the tests working on local... but it requires that OpenRA.Mods.Common.dll must also be located in the parent folder besides the Test dll. That's why the copying stuff...

The only way to solve this problem that occurs to my mind while keeping references non-private would be to change the Mods.Common project to also build in parent instead of bin/Debug.

Any suggestions about this?